packages/@aws-cdk/aws-lambda/lib/code.ts

```diff
@@ -160,6 +162,54 @@ export class InlineCode extends Code {
   }
 }
 
+/**
+ * Bundle options
+ */
+export interface BundleOptions {
+  /**
+   * The Docker image where the command will run.
+   */
+  readonly image: DockerImage;
+
+  /**
+   * The command to run in the container.
+   */
```

Comment:
Provide an `@example` here to demonstrate that this is a single command with arguments as array components